Extension/package.json

```diff
@@ -6197,6 +6224,18 @@
                     "title": "%c_cpp.codeActions.refactor.inline.macro.title%",
                     "description": "%c_cpp.codeActions.refactor.inline.macro.description%"
                 }
+            },
+            {
+                "languages": [
+                    "c",
+                    "cpp",
+                    "cude-cpp"
```

Comment:
typo: `cuda-cpp`

Reply:
@bobbrow Good catch, but FYI, I have not self-code review this PR yet, and it has not been cleaned up at all. So I wouldn't recommend reviewing it until it's ready. That's why it's in draft still.

Reply:
Don't worry, I just took a quick look because I'm motivated to get this feature in. I was looking at the outline more than anything and happened to see this.